packages/runtime/datastore/src/localChannelStorageService.ts

```diff
@@ -13,41 +13,41 @@ export class LocalChannelStorageService implements IChannelStorageService {
     }
 
     public async read(path: string): Promise<string> {
-        const contents = this.readSync(path);
-        return contents !== undefined ? Promise.resolve(contents) : Promise.reject(new Error("Not found"));
+        const blob = this.readBlobSync(path);
+        return blob !== undefined ? Promise.resolve(blob.contents) : Promise.reject(new Error("Not found"));
```

Comment:
blob.encoding is ignored here, and assumed utf-8. If that's indeed true, please add assert. If not, please add case for both base64 & utf-8.
I should be in sync with readBlob() in terms of supporting cases (readBlob supports both encodings)

Reply:
I'm not sure what read should return anymore. I thought it's base64 but when I saw the tests in localChannelStorageService.spec.ts, it returns utf8 string. Should I change the test?

Reply:
This code works with IBlob, so it should work correctly with either encoding - this is shipping code.
I'd guess that 
Test code provided encoding = utf-8, so it should keep working same was as is.

The problem here is not with utf-8 (it works correctly), but with base64 (it does not - it should decode it if encoding says so)